webpack.config.js

```diff
@@ -1,64 +1,56 @@
-const defaultsDeep = require('lodash.defaultsdeep');
```

Comment:
This file is slightly easier to view if you set "Hide whitespace" (or add `&w=1` to the URL)